domains/abby.json

```diff
@@ -5,6 +5,8 @@
         "discord": "699318362848886864"
     },
     "record": {
-        "CNAME": "mx.milosantos.com"
+        "CNAME": "mx.milosantos.com",
+        "MX": ["mx.milosantos.com"],
+        "TXT": "v=spf1 a mx include:mail.milosantos.com include:milosantos.com include:mail.abby.is-a.dev include:abby.is-a.dev -all"
```

Comment:
Checks are failing because of CNAME being together with other records. I'm not really sure how you can add all of these records to the file.

Reply:
Only think i can think of is to make a Pull request every time the IP changes (is pretty frequently) but i also dont wanna disturb a lot, not quite sure what to do besides that. @CuteDog5695

Reply:
Not sure if this will work, but maybe try to add all records to `mx.milosantos.com` and use only CNAME for `abby.is-a.dev`. Reason why I'm bringing up this idea is because CNAME takes all the records from the domain its pointed to.

Reply:
Yeah, there needs to be a way to allow her to get the domain pointed without having to make a PR every 15 days.

Reply:
I have an already configured & working server at `milosantos.com` but i think it wont work because of the DKIM verification using a specific KEY which is different from the `is-a.dev` domain's server @CuteDog5695

Reply:
I think we should issue you a NS record for your domain, that way you can manage the IPs and the domain without having to deal with a PR, and that would allow you to directly switch your IP without us seeing it.

Reply:
Honestly, at this point you might want to consider using NS records for `abby.is-a.dev`. If you are gonna do that, please delete all other subdomains along with adding NS records to the main file.

Reply:
> Thanks github for being slow as fuck on showing comments.

I have to reload every time you send a comment :sob:

Reply:
@DEV-DIBSTER @CuteDog5695 Alr, i delete all the subdomains and i delete all the records but add a NS record? I have a nameserver in ns1.milosantos.com and ns2.milosantos.com

Reply:
Yes. You add those NS records for the main file.

Reply:
> Yes. You add those NS records for the main file.

Alright, two NS records? or one?

Reply:
Two.

Reply:
We don't have a limit on how many NS records you need in one file I believe.

Reply:
#17402